### PR TITLE
Issue and PR comments for Bitbucket Cloud

### LINKED ISSUE #2856: Bitbucket Cloud: No comment with preview environment on pull request
### Summary 
When the preview environment of a Pull Request is ready, the pull request in Bitbucket Cloud doesn't receive a comment with the URL to the preview environment. Currently, the logs display the message "Bitbucket Cloud doesn't support adding PR comments via the REST API", but this now implemented: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D/comments

### Steps to reproduce the behavior

1. Commit a change to your codebase to a new Branch.
2. Create a pull request in Bitbucket Cloud
3. Check the build logs for the pull request in Jenkins.
4. Find "Bitbucket Cloud doesn't support adding PR comments via the REST API"

### Expected behavior

- No warning in the logs.
- A comment with the URL to the preview environment attached to your pull request in Bitbucket.

### Actual behavior

- No comment with the preview environment on the pull request.
- A warning in the logs: "Bitbucket Cloud doesn't support adding PR comments via the REST API"

### Jx version

The output of `jx version` is:

```
NAME               VERSION
jx                 1.3.798
jenkins x platform 0.0.3286
Kubernetes cluster v1.10.11-gke.1
kubectl            v1.10.11
helm client        v2.12.1+g02a47c7
helm server        v2.12.2+g7d2b0c7
git                git version 2.20.1.windows.1
Operating System   Windows 10 Pro 1803 build 17134
```

### Jenkins type
- [ X] Classic Jenkins
- [ ] Serverless Jenkins

### Kubernetes cluster
GKE Cluster, created with jx create cluster gke

### Operating system / Environment
Windows 10 Pro 1803 build 17134

### PR DESCRIPTION
**Pull request by wbrefvem**
#### Submitter checklist

- [x] Change is code complete and matches issue description.
- [x] Change is covered by existing or new tests.

#### Description
Introduces support for creating comments on issues and PRs on Bitbucket Cloud.

#### Which issue this PR fixes

fixes #2856

### CODE REVIEW

File: pkg/gits/bitbucket_cloud.go [hunk 1]
```diff
@@ -825,13 +820,40 @@ func (b *BitbucketCloudProvider) CreateIssue(owner string, repo string, issue *G
...
+		Content: &bitbucket.IssueContent{
+			Raw: comment,
+		},
+	}
+
+	_, _, err := b.Client.PullrequestsApi.RepositoriesUsernameRepoSlugPullrequestsPullRequestIdCommentsPost(
+		b.Context,
+		pr.Owner,
+		pr.Repo,
+		int32(*pr.Number),
```
Comment (ccojocar) [inline pkg/gits/bitbucket_cloud.go]: Is it worth checking if this pointer is not nil before the conversion?

Comment (wbrefvem) [reply pkg/gits/bitbucket_cloud.go]: Yeah that's a good point. I'll add a sanity check.

**Comment by wbrefvem** [comment]
@ccojocar I added a nil pointer check and rebased. Should be ready to go.

**Pull request merged**